src/Neo.CLI/config.fs.mainnet.json

```diff
@@ -40,7 +40,8 @@
       "HF_Aspidochelone": 3000000,
       "HF_Basilisk": 4500000,
       "HF_Cockatrice": 5800000,
-      "HF_Domovoi": 5800000
+      "HF_Domovoi": 5800000,
+      "HF_Echidna": 5800001
```

Comment:
@AnnaShaleva Should be removed from config

Reply:
It comes from the branch change

Reply:
Ping @AnnaShaleva

Reply:
Will update.

Reply:
Fixed.